.gitignore

```diff
@@ -199,3 +199,14 @@ FakesAssemblies/
 License.config
 /Blomsterringen.Web/imagecache
 maildrop
+.idea/.idea.Geta.DdsAdmin/riderModule.iml
```

Comment:
You can add to ignore list whole folder in that way avoiding to enlist each individual files